dotnet/test/VectorData/AzureAISearch.ConformanceTests/AzureAISearchDistanceFunctionTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using AzureAISearch.ConformanceTests.Support;
+using VectorData.ConformanceTests;
+using VectorData.ConformanceTests.Support;
+using Xunit;
+
+namespace AzureAISearch.ConformanceTests;
+
+public class AzureAISearchDistanceFunctionTests(AzureAISearchDistanceFunctionTests.Fixture fixture)
+    : DistanceFunctionTests<string>(fixture), IClassFixture<AzureAISearchDistanceFunctionTests.Fixture>
+{
+    public override Task CosineDistance() => Assert.ThrowsAsync<NotSupportedException>(base.CosineDistance);
+    public override Task NegativeDotProductSimilarity() => Assert.ThrowsAsync<NotSupportedException>(base.NegativeDotProductSimilarity);
+    public override Task EuclideanSquaredDistance() => Assert.ThrowsAsync<NotSupportedException>(base.EuclideanSquaredDistance);
+    public override Task HammingDistance() => Assert.ThrowsAsync<NotSupportedException>(base.HammingDistance);
+    public override Task ManhattanDistance() => Assert.ThrowsAsync<NotSupportedException>(base.ManhattanDistance);
+
+    public new class Fixture() : DistanceFunctionTests<string>.Fixture
+    {
+        public override string CollectionName => "distance-functions-" + AzureAISearchTestEnvironment.TestIndexPostfix;
+
+        public override TestStore TestStore => AzureAISearchTestStore.Instance;
+
+        public override bool AssertScores => false;
```

Comment:
Thanks for adding these, I've also been thinking we should fill in this gap, even for DBs that return non-standard scores.  I'm thinking it would be nice to call out that they are non-standard though, so that we know why we are not asserting scores here.
```suggestion
        // AzureAISearch does not return the expected standard mathematical result for each distance function.
        public override bool AssertScores => false;
```

Reply:
Sure, added the comment (though just noting that disabling the score assertion in itself kinda expresses that).